pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -268,7 +308,7 @@ func validateCompNameAndPathFolder(input interface{}) error {
 func SelectApplicationNameComp() string {
 	var applicationName string
 	prompt := &survey.Input{
-		Message: "Provide the Application name to add a Component",
+		Message: "Provide the Application name to add/delete a Component",
```

Comment:
I was referring to passing the string into the function, eg. 

func SelectApplicationNameComp(msg string) string {

So from the add command, msg will be "Provide the Application name to add a Component"